<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cache Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Project:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="projects_cmb">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Sequence:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sequences_cmb">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Shot:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="shots_cmb">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="exploreFolder_btn">
          <property name="text">
           <string>Open Pool Folder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="copyToServer_chb">
          <property name="text">
           <string>Copy To Server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="useShotFolder_chb">
          <property name="text">
           <string>Use Shot Folder</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refresh_btn">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>R</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="exportCamera_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 0);</string>
        </property>
        <property name="text">
         <string>Export Selected Camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="referenceCamera_btn">
        <property name="text">
         <string>Reference Camera And Time Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Caches</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QPushButton" name="exportSelectedGeo_btn">
          <property name="text">
           <string>Export Cache For Selected Geometry</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="exportAssetCache_btn">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 170, 0);</string>
            </property>
            <property name="text">
             <string>Export Cache For Selected Asset</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Steps:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="steps_spb">
            <property name="minimum">
             <double>0.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="connectToGlobalScale_chb">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Connect to global Scale</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="importShading_chb">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Import instead of reference it</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="useExocortex_chb">
            <property name="text">
             <string>Use Exocortex</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QTabWidget" name="caches_tw">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Anim</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QListWidget" name="animCaches_lw"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>SkinFix</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QListWidget" name="skinFixCaches_lw"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Sim</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QListWidget" name="simCaches_lw"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_6">
             <attribute name="title">
              <string>Vfx</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QListWidget" name="vfxCaches_lw"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Area to Ref/Imp:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="area_lw">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="currentRow">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string>Model</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Shading</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Hrs</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Rig</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Final</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="replaceAlembic_btn">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select AlembicNode in maya, and choose from UI the new alembic Node :)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Replace Cache</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Sets</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="exportSelectedToSet_btn">
          <property name="text">
           <string>Export Set To Pool</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="files_lw"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="loadExternalCache_btn">
        <property name="text">
         <string>Load External</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadSelectedCache_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Load Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QPushButton" name="exportAnimationScene_btn">
        <property name="text">
         <string>Export Animation Scene</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createLitScene_btn">
        <property name="text">
         <string>Create Lit Scene</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="swapShot_btn">
      <property name="text">
       <string>Swap Shot</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
